tests/tabular/test_data_model_integration.py

```diff
@@ -33,8 +33,8 @@ def test_classification(tmpdir):
     test_df = TEST_DF_1.copy()
     data = TabularData.from_df(
         train_df,
-        categorical_input=["category"],
-        numerical_input=["scalar_a", "scalar_b"],
+        cat_cols=["category"],
+        num_cols=["scalar_a", "scalar_b"],
```

Comment:
naming

Reply:
Not sure to understand your comment.

Reply:
same comment as @Borda before

Reply:
Yes, starting to add consistency in the names. And shorter here was better.